scripts/ci/_utils.sh

```diff
@@ -932,3 +935,17 @@ function build_image_on_ci() {
     unset AIRFLOW_CONTAINER_FORCE_PULL_IMAGES
     unset FORCE_BUILD
 }
+
+CHANGED_FILE_NAMES=$(git diff --name-only "HEAD...${TRAVIS_BRANCH:="HEAD"}")
```

Comment:
`CHANGED_PY_FILES=$(git diff HEAD --diff-filter=ACMRTUXB --name-only "$@" | grep -E '\.py$')`

That's what I was using in my pre commit hook before using the pre-commit framework for detecting python file changes. 

I was also using `diff-filter` because I passed the file names to pylint and in case the file that changed was deleted it won't work. But I think in this case here we do not need to filter.

So does the following work?
```suggestion
CHANGED_FILE_NAMES=$(git diff HEAD --name-only "$@" | grep -E '\.py$')
```

Reply:
👍 will check it.

Reply:
This only works for staged changes. In travis it works a bit differently apparently. Testing it now to have a temporary fix for kubernetes tests as well